packages/docs/src/pages/tutorial/component/inline/index.mdx

```diff
@@ -3,8 +3,12 @@ title: Light Component
 layout: tutorial
 ---
 
-Qwik components are lazy-loadable. This is achieved through `component$()` method. The presence of `$` crates a lazy-loaded boundary. The `$` will be discussed later. Qwik also supports inline components. Inline components are not by themselves lazy loadable and instead load as part of the parent component where they are used. (Inline components are equivalent to regular components in most other frameworks.)
+Qwik components are lazy-loadable, which lies at the heart of what makes Qwik different from most other frameworks.
 
-In this example we have pre-declared an `<App>` and a `<Greeter>` component. The `<Greeter/>` component is currently a Qwik Component because it is declared with `component$()`. Remove the `component$()` to convert `<Greeter>` to inline component. Inline components do not have host elements, (described later.)
+You can mark a component as lazy-loadable by using the `component$()` method. When you see a `$` in Qwik code, you're encountering a lazy-loading boundary.
```

Comment:
@craigshoemaker I would change the word "encountering" to "entering" like so -  "when you see a '$' in Qwik code, you are entering a lazy loaded boundry. "

@mhevery WDYT?

Reply:
@mhevery thanks!
Maybe it's a good point to mention the limitations of what "crossing the boundary" means? (to give more weight to it, so developers will know "this is an important point, I should make a mental note here")

WDYT?

Reply:
Yes! Every time you see `$`, you (as a developer) should be thinking special rules apply here. Not every valid JS is valid Qwik code.

Reply:
@mhevery do you think we should add these rules in this section as well?

Reply:
Hmm, they should probably be both here and in the https://qwik.builder.io/docs/advanced/optimizer!
```
- all captured variables must be declared `const`
- all captured variables must be either:
  - serializable
  - importable (either `import` or `export` in this file)
```

Reply:
Awesome!
Thanks @mhevery

Reply:
Here's #777 to add these rules to the Optimizer doc 😎